.github/workflows/snake.yml

```
@@ -0,0 +1,20 @@
+name: Generate contribution grid snake
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)